lib/hq/graphql/association_loader.rb

```diff
@@ -4,6 +4,7 @@ module HQ
   module GraphQL
     class AssociationLoader < ::GraphQL::Batch::Loader
       def initialize(model, association_name)
+        super()
```

Comment:
I just disabled new cops in our default rubocop config so this shouldn't be an issue anymore. Plesae remove.